src/pocketmine/tile/FlowerPot.php

```diff
@@ -66,12 +65,12 @@ public function canAddItem(Item $item) : bool{
 	}
 
 	public function getItem() : Item{
-		return Item::get((int) ($this->namedtag["item"] ?? 0), (int) ($this->namedtag["mData"] ?? 0), 1);
+		return Item::get($this->namedtag->item->getValue(), $this->namedtag->mData->getValue(), 1);
```

Comment:
It's possible for a plugin to unset them though. Unlikely: yes, but still possible